server/middlewares/ArticleValidation.js

```diff
@@ -184,6 +184,50 @@ class ArticleValidation {
     ).optional({ checkFalsy: false }).isAlpha();
   }
 
+  /**
+   * This method validates the comment id
+   * @param {object} req - The request object
+   * @returns {void}
+   */
+  static validateCommentId(req) {
+    req.checkParams(
+      'commentId',
+      'commentId must be an integer'
+    ).isInt();
+    req.checkParams(
+      'commentId',
+      'commentId must be greater than 0'
+    ).isInt({ gt: 0 });
+  }
+
+  /**
+   * This method validates the articleId
+   * @param {object} req - The request object
+   * @returns {void}
+   */
+  static validateArticleId(req) {
+    req.checkParams(
+      'articleId',
+      'articleId must be an integer'
+    ).isInt();
+    req.checkParams(
+      'articleId',
+      'articleId must be greater than 0'
+    ).isInt({ gt: 0 });
+  }
+
+  /**
+   * This method validates the comment reply
+   * @param {object} req - The request object
+   * @returns {void}
+   */
+  static validateCommentReply(req) {
+    req.checkBody(
+      'reply',
+      'please provide a reply'
+    ).exists();
```

Comment:
I think the` validateBody` method that already exists in this class could suffice for this check. You would just have to rename the the field `body` instead of reply and you can reuse it. Or is there a reason you have chosen to write this specifically?

Reply:
yes I think reply maybe better in this case.